### PR TITLE
TVAULT-4832 & TVAULT-4834 DevOps code to enable/disable encryption checkbox for RHOSP/TRIPLEO

### PR DESCRIPTION
**Pull request by trilio-shiwanand**
local_settings file is mounted under horizon container from controller node(s)
The setting defined under local_settings got reflected on the service

### CODE REVIEW

**Review by shyam-biradar — APPROVED** [review]

File: redhat-director-scripts/rhosp16.1/puppet/trilio/manifests/horizon.pp [hunk 1]
```diff
@@ -1,15 +1,34 @@
...
+       }
+   }
 
-}
\ No newline at end of file
+
+    file_line { "ENABLE TRILIO_ENCRYPTION_SUPPORT":
+        ensure => present,
+        path   => "${horizon_dir}/local_settings",
+        line   => 'TRILIO_ENCRYPTION_SUPPORT = True',
```
Comment (DeepikaAFour) [inline redhat-director-scripts/rhosp16.1/puppet/trilio/manifests/horizon.pp:29]: when the variable set to False?

File: redhat-director-scripts/rhosp13/puppet/trilio/manifests/horizon.pp [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
+	   line   => 'OPENSTACK_ENCRYPTION_SUPPORT = False',
+	   match  => '^OPENSTACK_ENCRYPTION_SUPPORT',
+       }
+   }
+ 
+
+    file_line { "DISABLE TRILIO_ENCRYPTION_SUPPORT":
+        ensure => present,
+        path   => "${horizon_dir}/local_settings",
+        line   => 'TRILIO_ENCRYPTION_SUPPORT = False',
```
Comment (DeepikaAFour) [inline redhat-director-scripts/rhosp13/puppet/trilio/manifests/horizon.pp:29]: when the variable set to True?

File: redhat-director-scripts/rhosp16.1/puppet/trilio/manifests/horizon.pp (hunk 1 above)
Comment (trilio-shiwanand) [reply redhat-director-scripts/rhosp16.1/puppet/trilio/manifests/horizon.pp:29]: barbican_api_enabled get's false in that case  OPENSTACK_ENCRYPTION_SUPPORT value set to the False

File: redhat-director-scripts/rhosp13/puppet/trilio/manifests/horizon.pp (hunk 2 above)
Comment (trilio-shiwanand) [reply redhat-director-scripts/rhosp13/puppet/trilio/manifests/horizon.pp:29]: For RHOSP13 TRILIO_ENCRYPTION_SUPPORT we are not supporting.
Hence in the RHOSP13 TRILIO_ENCRYPTION_SUPPORT Default value is set to the False.

Comment (DeepikaAFour) [reply redhat-director-scripts/rhosp13/puppet/trilio/manifests/horizon.pp:29]: please add this in the code documentation.

File: redhat-director-scripts/rhosp16.1/puppet/trilio/manifests/horizon.pp (hunk 1 above)
Comment (DeepikaAFour) [reply redhat-director-scripts/rhosp16.1/puppet/trilio/manifests/horizon.pp:29]: I don't see the False setting code in `if barbican_api_enabled` block.
and the TRILIO_ENCRYPTION_SUPPORT = True is also not depend on any condition.
Am I missing anything here?

**Pull request merged**